Source/Modules/ModuleEditor.h

```diff
@@ -25,6 +25,7 @@ class ModuleEditor : public Module
 	void Resized();
 
 	const WindowScene* GetScene() const;
+	const std::unique_ptr<WindowDebug>* GetDebugOptions() const;
```

Comment:
This is a pointer to a unique pointer, which doesn't make sense. 
```suggestion
const std::unique_ptr<WindowDebug> GetDebugOptions() const;
```
All the calls to `get()` can be removed as well. So, instead of doing `App->editor->GetDebugOptions()->get()->...` you should do `App->editor->GetDebugOptions()->...`